user-guide/docs/usecases/workinprogress.md

```diff
@@ -0,0 +1,18 @@
+# Work in Progress
+
+The documentation in this section is still a work in progress. <br>
+It has been published here for testing purposes.
+
+<!-- ##  CFD Analysis of Winds on Low-Rise Building -->
+
+{% include-markdown 'kareem/usecase2.md' %}
+
+<hr style="border: dashed white 1.0px;background-color: white;height: 200.0px;"/>
+---
+
+<!-- ##  Tamkang Database -->
+
+{% include-markdown 'kareem/usecase3.md' %}
+
+<hr style="border: dashed white 1.0px;background-color: white;height: 200.0px;"/>
```

Comment:
This solution has drawbacks —

- redundant `<hr>`s
- duplicate complex code
- inline styles

— so please [use a stylesheet instead](https://github.com/DesignSafe-CI/DS-User-Guide/pull/18).

<sup>To learn more, see "Notes" on [#18](https://github.com/DesignSafe-CI/DS-User-Guide/pull/18).</sup>